### DISCUSSION TITLE
外贸小助手

### DISCUSSION THREAD

**Discussion by flh1107** [InternLM/Tutorial#400] [project]
项目目标：
1、搭建进出口贸易相关的大模型，收集公告、国家发布数据等公开语料进行训练
2、研究大模型在进出口贸易走势预测、智能化咨询等方面的运用场景

**Comment by JinProton** [comment]
你好，这个还缺人吗

**Comment by burningmysoul2077** [comment]
您好，本人从事航运领域，LLM小白但很想学习，可以加入您的队伍吗？
